packages/typespec-vscode/src/tsp-language-client.ts

```diff
@@ -239,10 +239,12 @@ export class TspLanguageClient {
     srcFolder: string,
     outputFolder: string,
   ): Promise<ExecOutput | undefined> {
+    // If the path contains spaces, it will cause compilation failure, so quotes are required
+    const escapedMainTspFile = mainTspFile.includes(" ") ? `"${mainTspFile}"` : mainTspFile;
     const result = await this.runCliCommand(
       [
         "compile",
-        mainTspFile,
+        escapedMainTspFile,
```

Comment:
this feels like the job of whatever is happening in `runClieCommand` to deal with this and escape the parameters correctly no?

Reply:
look at using `cross-spawn` that solve this issue